doc/modules/feature_selection.rst

```diff
@@ -131,33 +131,50 @@ number of features.
       elimination example with automatic tuning of the number of features
       selected with cross-validation.
 
+.. _select_from_model:
 
-.. _l1_feature_selection:
+Feature selection using SelectFromModel
+=======================================
+
+:class:`SelectFromModel` is a meta-transformer that can be used along with any
+estimator that has a ``coef_`` or ``feature_importances_`` attribute after fitting.
+It should be given a ``threshold`` parameter below which the features are considered
+unimportant and removed. If one has no idea of the prior value of the threshold,
```

Comment:
Apart from specifying the threshold numerically, there are build-in heuristics for finding a threshold, which can be selected using a string argument. Available heuristics are ``"mean``", ``"median"`` and  float multiples of these like ``"0.1*mean"``. [maybe something along these lines?]